porch/config/samples/create-deployment-package.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
header?

Reply:
Good point - we need to get this under testing - I have some ideas :-)